contextproject-ms3/src/main/java/contextproject/export/M3UExport.java

```diff
@@ -0,0 +1,82 @@
+package contextproject.export;
+
+import contextproject.models.Playlist;
+import contextproject.models.Track;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.PrintWriter;
+
+public class M3UExport {
+
+  private Playlist playlist;
+  private String playlistName;
+  private String pathName;
+
+  /**
+   * Constructor of the export.
+   * 
+   * @param pl
+   *          playlist.
+   * @param name
+   *          name of playlist.
+   */
+  public M3UExport(Playlist pl, String name) {
+    playlist = pl;
+    playlistName = name;
+    createPathName();
+  }
+
+  /**
+   * export method.
+   */
+  public void export() {
+    File file = new File(pathName);
+
+    try {
+      PrintWriter writer = new PrintWriter(file);
+      for (Track track : playlist) {
+        writer.println(track.getPath());
+      }
+      writer.close();
+    } catch (FileNotFoundException e) {
+      System.out.println("File doesn't excist.");
+    }
+  }
```

Comment:
I think it's a good idea to separate the string building logic and the saving of m3u file, because we might have more exporter classes in the future. This way it's more modular / easier to test.